aioamqp/compat.py

```diff
@@ -1,11 +0,0 @@
-"""
-    Compatibility between python or package versions
-"""
-# pylint: disable=unused-import
-
-import asyncio
-
-try:
-    from asyncio import ensure_future
-except ImportError:
-    ensure_future = getattr(asyncio, 'async')
```

Comment:
nope, still needed for python 3.4.2 (which we want to support for a little while longer)

Reply:
Thanks for the feedback.  I missed that by checking only Python 3.4.8 :/